core/templates/pages/topics-and-skills-dashboard-page/delete-topic-modal.controller.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for the delete topic confirmation modal.
+ */
+
+angular.module('oppia').controller('DeleteTopicModalController', [
```

Comment:
I think you should use ConfirmOrCancelModalController here. It was added in an attempt to reduce code duplication for modals and this should also reduce the number of tests you need to write. @marianazangrossi can help you out if needed (she just did a migration where she used the ConfirmOrCancelModalController across all existing modals with just a confirm and cancel button).

Reply:
Done.